<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Matriel</class>
 <widget class="QWidget" name="Matriel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1308</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>1189</x>
     <y>598</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>351</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Ajouter/Modifier un  Materiel</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>33</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>idMAT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>73</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>nomMAT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantite</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idMat">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nomMat">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Quantite">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>151</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Disponibilite</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Dispo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="idMAT_Delete">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="rechID">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>470</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Rechercher par ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ajoutMat">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="recherch_id">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>490</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="afficherMar">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>460</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Affichage</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Rtri">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>480</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>tri par % Quantite</string>
   </property>
  </widget>
  <widget class="QTableView" name="tabMAT">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>50</y>
     <width>721</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="supprimerMat">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>470</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Rtri_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>500</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>tri par % Nom Matricule </string>
   </property>
  </widget>
  <widget class="QPushButton" name="modifierMat">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>330</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
